examples/example-xds/README.md

```diff
@@ -12,40 +12,60 @@ environment.__
 
 ### Build the example
 
-1. Build the hello-world example server or the hostname example server. See
-   [the examples README](../README.md) or the
-   [hostname example README](../example-hostname/README.md).
-
-2. Build the xds hello-world example client. From the `grpc-java/examples/examples-xds` directory:
+Build the XDS hello-world example client & server. From the `grpc-java/examples/examples-xds`
+directory:
 ```
 $ ../gradlew installDist
 ```
 
-This creates the script `build/install/example-xds/bin/xds-hello-world-client`
-that runs the example.
+This creates the scripts `build/install/example-xds/bin/hello-world-client-xds` and
+`build/install/example-xds/bin/hello-world-server-xds`.
+
+### Run the example without using XDS Credentials
 
-To start the server, run:
+To use XDS, you should first deploy the XDS management server in your deployment environment
+and know its name. You need to set the `GRPC_XDS_BOOTSTRAP` environment variable to point to the
+gRPC XDS bootstrap file (see
+[gRFC A27](https://github.com/grpc/proposal/blob/master/A27-xds-global-load-balancing.md#xdsclient-and-bootstrap-file) for the
+bootstrap format). This is needed by both `build/install/example-xds/bin/hello-world-client-xds`
+and `build/install/example-xds/bin/hello-world-server-xds`.
 
+1. To start the XDS-enabled example server, run:
 ```
-$ ../build/install/hostname/bin/hello-world-server
-$ # or
-$ ../example-hostname/build/install/hostname/bin/hostname-server
+$ export GRPC_XDS_BOOTSTRAP=/path/to/bootstrap.json
+$ ./build/install/example-xds/bin/hello-world-server-xds 8000 my-test-xds-server
 ```
 
-And in a different terminal window run this client:
+The first command line argument is the port to listen on (`8000`) and the second argument is a string
+id (`my-test-xds-server`) to be included in the greeting response to the client.
 
+2. In a different terminal window, run the XDS-enabled example client:
```

Comment:
Does this work for localhost? Probably should say on the client machine/VM, etc.

Reply:
I don't see why it won't work for localhost (of course the routing will happen via xDS which will send the localhost's backend address to the client). Our interop xds tests work that way